earthly/cassandra/Earthfile

```diff
@@ -0,0 +1,39 @@
+# cspell: words scylladb ensurepath
+
+VERSION 0.8
+
+IMPORT ../../utilities/cql-to-d2 AS cql-to-d2-utils
+
+scylladb-base:
+    FROM scylladb/scylla:6.1.1
+
+    WORKDIR /root
+
+    RUN apt-get update && apt-get install -y python3 pipx
+
+    RUN pipx ensurepath
+    RUN pipx install poetry
+
+# cql-to-d2 - converts cql files into d2 diagram entity files
+cql-to-d2:
+    FROM +scylladb-base
+
+    ARG --required input
+    ARG --required output
```

Comment:
I think we can get rid of `output` argument here and define a concrete output place.
So it will reduce a headache by providing this output.